fv3core/stencils/remap_profile.py

```diff
@@ -297,23 +294,17 @@ def set_inner_as_kord9(
         tmp_min = a4_1
         tmp_max = a4_2
         tmp_max0 = a4_1
-        diff_23 = 0.0
-        if extm and extm[0, 0, -1]:
-            a4_2 = a4_1
-            a4_3 = a4_1
-            a4_4 = 0.0
-        elif extm and extm[0, 0, 1]:
-            a4_2 = a4_1
-            a4_3 = a4_1
-            a4_4 = 0.0
-        elif extm > 0.0 and (qmin > 0.0 and a4_1 < qmin):
+        if (
+            (extm != 0.0 and extm[0, 0, -1] != 0.0)
+            or (extm != 0.0 and extm[0, 0, 1] != 0.0)
+            or (extm > 0.0 and (qmin > 0.0 and a4_1 < qmin))
+        ):
```

Comment:
sweet!

Reply:
Yes, the conditionals have come a long way!